docs/index-shared1.asciidoc

```diff
@@ -15,6 +7,7 @@ release-state can be: released | prerelease | unreleased
 :blob:           {repo}blob/{branch}/
 :security-ref:   https://www.elastic.co/community/security/
 
+include::{asciidoc-dir}/../../shared/versions55.asciidoc[]
```

Comment:
In order for these values to be accessible when creating the above kibana-specific variables like `:docker-image:`, which relies on `{version}`, I think this include needs to happen at the top of the file.